lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java

```diff
@@ -33,55 +33,46 @@ public static void GlobalTearDown() {
     }
 
     @Test
-    public void sum() {
-        int a = 4;
-        int b = 3;
-
-        int expected = 7;
-        int actual = calculator.sum(a, b);
-
+    public void testDivision() {
+        int a = 12;
+        int b = 4;
+        int expected = 3;
+        int actual = calculator.division(a,b);
```

Comment:
use sonar, spaces should be added

Reply:
I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files.
I had added spaces, but sonar tests still failed. Help me, please.